src/lib/workspace/DownloadTab/PastRelease.tsx

```diff
@@ -0,0 +1,129 @@
+import { useMemo, useState } from 'react';
+import { Column } from 'react-table';
+
+// Components
+import { colors, DataGrid, Download } from '@veupathdb/coreui';
+
+// Definitions
+import { DownloadClient } from '../../core/api/DownloadClient';
+import { DownloadTabStudyRelease } from './types';
+
+// Hooks
+import { ReleaseFile, useGetReleaseFiles } from './hooks/useGetReleaseFiles';
+import { ExpandablePanel } from '@veupathdb/coreui/dist/components/containers';
+
+export type PastReleaseProps = {
+  studyId: string;
+  release: DownloadTabStudyRelease;
+  downloadClient: DownloadClient;
+};
+
+export default function PastRelease({
+  studyId,
+  release,
+  downloadClient,
+}: PastReleaseProps) {
+  const [releaseFiles, setReleaseFiles] = useState<Array<ReleaseFile>>([]);
+  const memoizedReleaseFiles = useMemo(() => releaseFiles, [releaseFiles]);
+
+  useGetReleaseFiles(studyId, release, downloadClient, setReleaseFiles);
+
+  const exampleGridColumns: Array<Column> = [
+    {
+      Header: 'File Description',
+      accessor: (row: any) => ({
+        description: row.fileDescription,
+        url: row.fileUrl,
+        name: row.fileName,
+      }),
+      Cell: ({ value }) => {
+        return (
+          <a
+            style={{
+              display: 'flex',
+              alignItems: 'center',
+              cursor: 'pointer',
+              color: colors.mutedCyan[500],
+            }}
+            href={value.url}
+          >
+            <Download fill={colors.mutedCyan[500]} fontSize={20} />
+            <span
+              style={{
+                fontFamily: 'Inter',
+                fontWeight: 600,
+                letterSpacing: 0.1,
+                marginLeft: 8,
+                marginRight: 25,
+              }}
+            >
+              {value.description}
+            </span>
+          </a>
+        );
+      },
+    },
+    {
+      Header: 'Type',
+      accessor: 'fileType',
+    },
+    {
+      Header: 'Size',
+      accessor: 'fileSize',
+      Cell: ({ value }) => <span>{value} MB</span>,
+    },
+  ];
+
+  return (
+    <div id="Current Release Dataset" style={{ marginBottom: 35 }}>
```

Comment:
Oops! I guess this `id` should be something `Past Release Datasets`.

Reply:
It's just nice when you are using dev tools and going through the component hierarchy during debugging.

Reply:
TIL!